packages/generator-single-spa/src/react/templates/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I think it might make sense to make a `single-spa-react-ts-config` npm package and then have generator-single-spa-react create a ts config that extends that one.

But let's do that in a future PR.